tests/integ/test_neo_mxnet.py

```diff
@@ -71,10 +72,16 @@ def test_attach_deploy(
             output_path=estimator.output_path,
         )
 
+        serializer = JSONSerializer()
+        serializer.CONTENT_TYPE = "application/vnd+python.numpy+binary"
```

Comment:
The sample notebooks demonstrating Neo uses a serializer similar to `NumpySerializer`, but when I tried `NumpySerializer` I ran into errors. 

The code above mimics the original behavior of the test. After doing more research on how Neo works, I can introduce the appropriate serializer